src/lv_api_map_v9_1.h

```diff
@@ -81,7 +81,15 @@ extern "C" {
 #define lv_chart_set_all_value lv_chart_set_all_values
 #define lv_calendar_set_showed_date lv_calendar_set_month_shown
 
-#define LV_FS_DEFAULT_DRIVE_LETTER     LV_FS_DEFAULT_DRIVER_LETTER
+#if defined(LV_FS_DEFAULT_DRIVE_LETTER)
+#warning LV_FS_DEFAULT_DRIVE_LETTER is deprecated. Rename to LV_FS_DEFAULT_DRIVER_LETTER
+
+#if LV_FS_DEFAULT_DRIVER_LETTER == '\0'
+#define LV_FS_DEFAULT_DRIVER_LETTER LV_FS_DEFAULT_DRIVE_LETTER
```

Comment:
```suggestion
#undef LV_FS_DEFAULT_DRIVER_LETTER
#define LV_FS_DEFAULT_DRIVER_LETTER LV_FS_DEFAULT_DRIVE_LETTER
```

Undef first to avoid "`warning: "LV_FS_DEFAULT_DRIVER_LETTER" redefined`" unless you think it's good to have.

Reply:
It would probably cause too much confusion so I agree it's better to add the undef clause